src/lib.rs

```diff
@@ -144,6 +148,28 @@ impl Builder {
             timeout: None,
             headers: HashMap::new(),
             max_retries: DEFAULT_MAX_RETRIES,
+            runtime: PhantomData,
+        }
+    }
+
+    /// Build a blocking client from builder
+    #[cfg(feature = "blocking")]
+    pub fn build_blocking(self) -> BlockingClient {
+        BlockingClient::from_builder(self)
+    }
+}
+
+impl<S: Sleeper> Builder<S> {
```

Comment:
Should we move the methods `proxy`, `timeout`, `header` and `max_retries` to the above `impl Builder` with no additional bounds on `S`?